train_config/training_server/image_classification/classification.py

```diff
@@ -45,8 +48,11 @@ def train_img_classification(config):
     # Train model
     model = fit_model(config, model, train_loader, val_loader, device, checkpoint_path)
 
-    # Save model
-    save_model(os.path.join(checkpoint_path, 'model.pt'), target_model_path)
+    # Save pt model
+    save_model_pt(os.path.join(checkpoint_path, 'model.pt'), target_model_path_pt)
+
+    # Save onnx model
+    save_model_onnx(os.path.join(checkpoint_path, 'model.pt'), target_model_path_onnx)
```

Comment:
Change extension name

Reply:
This is not an error. We are converting to onnx model from .pt model.